judge/utils/tests/test_two_factor.py

```diff
@@ -0,0 +1,21 @@
+from django.test import SimpleTestCase
+
+from judge.utils.two_factor import webauthn_decode, webauthn_encode
+
+
+class TwoFactorTestCase(SimpleTestCase):
+    def test_webauthn_decode(self):
+        self.assertEqual(webauthn_decode(''), b'')
+        self.assertEqual(webauthn_decode('AA'), bytes(range(1)))
+        self.assertEqual(webauthn_decode('AAE'), bytes(range(2)))
+        self.assertEqual(webauthn_decode('AAEC'), bytes(range(3)))
+        self.assertEqual(webauthn_decode(
+            'AAECAwQFBgcICQoLDA0ODxAREhMUFRYXGBkaGxwdHh8gISIjJCUmJygpKissLS4vMDEyMzQ1Njc4OTo7PD0-Pw'), bytes(range(64)))
```

Comment:
Should probably exercise `_` handling...

Reply:
Done